### ISSUE TITLE
Letter spacing is not supported

### ISSUE THREAD

**Issue by maximilientyc** [vadimdemedes/tailwind-rn#28]
To reproduce, simply create a new `Text` element with `tracking-tighter` class:

```xml
<Text
   style={tailwind(
     'tracking-widest'
   )}>
  Hello World!
</Text>
```

And you'll have the following error:
```
Warning: Failed prop type: Invalid prop `letterSpacing` of type `string` supplied to `Text`, expected `number`.
Bad object: {
  "letterSpacing": "0.1em"
}
```

I can create a PR to update the `README` to update the supported utilities, just let me know. If you have troubles reproducing the issue, I would be happy to help also.

Workaround: customize tailwind config and override value in `px`.

Max

**Comment by vadimdemedes** [comment]
Thanks for reporting, probably got broken in the latest Tailwind CSS releases.

**Comment by jtstodola** [comment]
You need to pass the key value pair inside of the letterSpacing object within the tailwind config.
```
const { letterSpacing } = require('tailwindcss/defaultTheme')

letterSpacing: {
  ...letterSpacing,
  tightest: '-0.125rem',
},
```

**Comment by vadimdemedes** [comment]
Released in [2.0.0](https://github.com/vadimdemedes/tailwind-rn/releases/tag/v2.0.0).

**Comment by maximilientyc** [comment]
Thank you!
